# Fluent for task planner

# name of the fluent
string name

# list of predicate params
diagnostic_msgs/KeyValue[] params

# fluent value
string value
uint8 value_type

# constants for type
uint8 STRING = 0
uint8 INT = 1
